rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/DiscoverHibernateConfigurationRuleProvider.java

```diff
@@ -141,14 +142,19 @@ private void createHibernateConfigurationModel(GraphRewrite event, EvaluationCon
 
                 DataSourceModel dataSource = dataSourceService.createUnique(applications, dataSourceName, dataSourceJndiName);
 
-                if (sessionFactoryProperties.containsKey("hibernate.dialect"))
+                if (sessionFactoryProperties.containsKey("dialect"))
```

Comment:
I still see "hibernate.dialect" in the docs as well:
https://docs.jboss.org/hibernate/orm/3.6/reference/en-US/html/session-configuration.html